vaadin-core/pom.xml

```diff
@@ -42,6 +42,10 @@
             <groupId>com.vaadin</groupId>
             <artifactId>fusion-endpoint</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-dev-server</artifactId>
```

Comment:
is this artifact available now ?
didnt find it in the snapshot repo http://tools.vaadin.com/nexus/content/repositories/vaadin-prereleases/com/vaadin/

Reply:
oh, still running https://bender.vaadin.com/viewLog.html?buildId=233042&tab=buildResultsDiv&buildTypeId=Flow_Flow80_Flow80Snapshot